react/features/invite/components/InviteButton.web.js

```diff
@@ -87,6 +88,11 @@ class InviteButton extends Component {
      * @returns {ReactElement}
      */
     render() {
+        // hide it in iAmRecorder mode
+        if (config.iAmRecorder) {
+            return null;
+        }
+
```

Comment:
It makes sense to me for the parent of the component to control visibility of the child, by either attaching or detaching. I do not believe the InviteButton needs to be concerned with its own visibility. This can be done in where the InviteButton is used, in Filmstrip.